.github/workflows/new-spec-release.yml

```diff
@@ -0,0 +1,103 @@
+name: Update spec document because of new spec release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with latest spec release
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v2
+        with:
+          path: spec
+          ref: ${{ github.event.release.target_commitish }}
+      - name: Checkout Another repository
+        uses: actions/checkout@v2
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b spec-release-${{github.event.release.tag_name}}
+      - name: Check for previous spec file and remove it
+        uses: actions/github-script@v3
+        if: ${{github.event.release.prerelease == false}}
+        with:
+          github-token: ${{ env.GITHUB_TOKEN }}
+          script: |
+            const fs = require("fs");
+
+            const specFiles = fs.readdirSync("./website/pages/docs/specifications");
+
+            const nextRelease = `v${{github.event.release.tag_name}}`;
```

Comment:
no need to append `v`. Previous releases didn't have it, but next ones will, I even added v2.0.0 tag recently,